trade-finance-manager-api/src/v1/services/user.service.upsert-user-from-entra-id-user.test.ts

```
@@ -80,7 +82,7 @@ describe('user service', () => {
       });
 
       it('throws an error', async () => {
```

it('should throw an error